openapi/golem-service.yaml

```diff
@@ -4544,19 +4544,6 @@ components:
           type: string
         invocationContext:
           type: string
-        allowOrigin:
```

Comment:
This a leftover from a different pr. @afsalthaj was this intentional?

After this pr I'll open a small followup adding an up-to-date check for the openapi.yaml to our ci

Reply:
I hope you resolved this doubt yourself recently :)